R/bayesianStateSpace.R

```diff
@@ -810,3 +810,4 @@ quantInv <- function(distr, value){
 
   return()
 }
+
```

Comment:
fyi: you can use `git commit --allow-empty -m "trigger tests"` instead of adding a whitespace

Reply:
cool I didn't know about that!